docs/src/main/sphinx/release/release-461.md

```diff
@@ -0,0 +1,43 @@
+# Release 461 (9 Oct 2024)
+
+## General
+
+* Rename the configuration property `max-tasks-waiting-for-execution-per-stage`
+  to `max-tasks-waiting-for-execution-per-query` and the session property
+  `max_tasks_waiting_for_node_per_stage` to
+  `max_tasks_waiting_for_node_per_query` to match implemented semantics. ({issue}`23585`)
+* Fix failure when joining tables with large numbers of columns. ({issue}`23720`)
+
+## Security
+
+* Add support for BCrypt versions 2A, 2B, and 2X. ({issue}`23648`)
+
+## Web UI
+
+* Add buttons to the query list to access details. ({issue}`22831`)
+* Brighten color scheme and adjust to Trino branding. ({issue}`22831`)
```

Comment:
We don't need to mention this one.

Reply:
Is VERY visible to users though

Reply:
It’s visible, but it’s not a change in behavior that matters to users. I think of it in the same category as doc improvements or changing error messages.

Reply:
Hm .. I guess ... I will remove it.